etc/custom.properties

```diff
@@ -146,7 +146,10 @@ org.opencastproject.db.ddl.generation=true
 
 ######### Paella player configuration folder #########
 
-org.opencastproject.engage.paella.config.folder=${karaf.etc}/paella
+# Path to the Paella player configuration files. Note that the configuration file for each tenant needs to be in a
+# subfolder with the tenants identifier, e.g. …/etc/mh_default_org/config.json
```

Comment:
Hmm... is that multi-tenant stuff really true? I can't see the logic that locates the file in a subfolder in etc....

Reply:
I can ;-P
It's in the REST endpoint.